source/declarations.tex

```diff
@@ -5425,27 +5425,16 @@
 \indextext{UTF-8}%
 \indextext{UTF-16}%
 \indextext{UTF-32}%
-An array of ordinary character type\iref{basic.fundamental},
```

Comment:
I think there is a semantics change relative to the old wording here. The previous wording would have allowed for `unsigned char[]` to be initialized by an ordinary string literal; the new wording does not.

Reply:
Fixed in the latest wording.